src/katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -611,6 +611,23 @@
             "additionalProperties": false,
             "properties": {
                 "develop": {"type": "boolean", "default": false},
+{% if version >= "3.2" %}
+                "develop": {
+                    "oneOf": [
+                        {"type": "boolean"},
```

Comment:
```suggestion
                        {"type": "boolean", "default": false},
```

Reply:
Done, but I now have to move the `,` to inside the `oneOf` statement, not quite sure how to format this  ?

Reply:
What I think looks the nicest is to swap the order of the two branches of the oneOf, which lets the comma sit in its natural place. But what you've done is fine too.

Reply:
Thanks, I did consider that as an option. I've changed it.